source/kaleidic/lubeck2.d

```diff
@@ -1280,12 +1280,14 @@ Returns: error code from CBlas
             }
         }
         enum msg = (algorithm == "gesvd" ? "svd: DBDSDC did not converge, updating process failed" : "svd: DBDSQR did not converge");
-        enforce!("svd: " ~ msg)(!info);
+        
+        try enforce!("svd: " ~ msg)(!info);
+        catch(Exception e) assert(false, e.msg);
```

Comment:
SVF isn't supposed to be nothrow with the current API. You may want to add an additoinal low-level API that will incorporate the info into the result.